src/main/java/servlets/ResultsServlet.java

```diff
@@ -66,24 +66,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(gson.toJson(testRecipes));
   }
 
-  public List<String> formatQueryAsList(String query) {
-    query = query.toUpperCase();
-    List<String> queryList = new ArrayList<String>(Arrays.asList(query.split(",")));
-    List<String> formattedQueryList = new ArrayList<String>();
-    for (String singleQuery:queryList) {
-      formattedQueryList.add(singleQuery.trim());
-    }
-    return formattedQueryList;
+  public String[] formatQueryAsList(String query) {
```

Comment:
Why do you need to change this?

Reply:
https://github.com/googleinterns/step20-2020/pull/15#discussion_r454763059

Reply:
Thanks.